src/core/components/Alert.tsx

```diff
@@ -5,40 +5,24 @@ import {
   XCircleIcon,
 } from "@heroicons/react/24/outline";
 import { useTranslation } from "next-i18next";
-import { ReactNode, useEffect, useMemo, useState } from "react";
+import { ReactNode, useEffect, useState } from "react";
 import Button from "./Button/Button";
 import Dialog from "./Dialog";
-import { AlertType } from "core/helpers/alert";
 
 type AlertProps = {
-  type?: AlertType;
+  icon: ReactNode;
   onClose: () => void;
   children: ReactNode;
 };
 
-const Alert = (props: AlertProps) => {
-  const { type, children, onClose } = props;
+const Alert = ({ icon, children, onClose }: AlertProps) => {
```

Comment:
I am keep `Alert` component because having a blocking dialog replacing the content is a different use case than toasts.